### ISSUE TITLE
getcarriers=1 not working on textbelt.com/text

### ISSUE THREAD

**Issue by djds23** [typpo/textbelt#64]
When I clone & run the project locally, everything appears to be a-okay. The endpoint returns an array of carriers:
<img width="600" alt="screen shot 2015-12-30 at 6 15 22 pm" src="https://cloud.githubusercontent.com/assets/5833968/12059033/934abdc0-af21-11e5-8def-321f5c4f3308.png">

However in production when I issue the same command I just get the  `{"success": true}` response:
<img width="516" alt="screen shot 2015-12-30 at 6 15 51 pm" src="https://cloud.githubusercontent.com/assets/5833968/12059034/a09ff238-af21-11e5-8bb8-f9320c851327.png">

Command in question:
```shell
curl -X POST http://textbelt.com/text \
	-d "getcarriers=1" \
	-d "number=2025550106" \
	-d "message=I sent this message for free with textbelt.com"
```
Is it possible the code running on `textbelt.com` is out of date?

As a note the number I've provided is fake.

**Comment by typpo** [comment]
Yeah, the current textbelt.com hasn't been updated with the carriers code, which was merged a bit under a month ago.  https://github.com/typpo/textbelt/pull/57

I haven't updated the server because the carriers stuff wasn't in the documentation.  I intended to change the endpoint to something like `/carriers/`and update the documentation accordingly, but haven't done so yet.
